files/default/slack_handler_webhook.rb

```diff
@@ -96,4 +110,20 @@ def request_body(message, text_attachment)
   def run_status_human_readable
     run_status.success? ? "succeeded" : "failed"
   end
+
+  def run_status_cookbook_detail(detail_level)
+    detail_level ||= @cookbook_detail_level
+    case detail_level
+    when "off"
```

Comment:
Updated in both usages - just removed the defaults complete for when `nil` needs to be returned.